sync.py

```diff
@@ -282,19 +282,33 @@ def get_playlists_from_spotify(spotify_session, config):
             break
     return playlists
 
-def get_playlists_from_config(config):
+def get_playlists_from_config():
     # get the list of playlist sync mappings from the configuration file
-    return [(item['spotify_id'], item['tidal_id']) for item in config['sync_playlists']]
+    return [(item['spotify_id'], item['tidal_id']) for item in ConfigHolder.get_config()['sync_playlists']]
+
+class ConfigHolder:
```

Comment:
I think a class that is never instantiated like this is a bit weird and not very pythonic. Probably better to use a config module with an importable config variable like this:

https://codereview.stackexchange.com/a/269570

Reply:
Probably better to sync with @joshrmcdaniel and wait for his changes to merge before doing this however:
https://github.com/spotify2tidal/spotify_to_tidal/pull/29#pullrequestreview-2042312916